tests/helper/tests/systemctl.py

```diff
@@ -1,90 +1,27 @@
-import logging
+def systemctl(client, state, services):
+    """Test to check if systemd services are enabled or disabled as desired"""
+    # get list of enabled systemd services
+    if state == "enabled":
+        (exit_code, output, error) = client.execute_command(
+            "systemctl list-unit-files | awk '$2~/static/ { print $1; " +
```

Comment:
Just a suggestion - I'd rather use a more Pythonic way by interacting with `output` instead of these shell tools.